src/code-templates/libraries/configuration-provider/package.json

```diff
@@ -1,10 +1,12 @@
 {
-  "name": "configuration-provider",
-  "version": "0.0.1",
+  "name": "@practica/configuration-provider",
+  "version": "0.0.1-alpha.1",
   "description": "",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
I think it will be good if we release a "clean" version in the future - 
1. no _.ts_ files
2. _index.js_ inside _root_, not _dist_

open a ticker for the future maybe?

Reply:
Yes!